legal-api/src/legal_api/resources/v2/business/business_comments.py

```diff
@@ -46,14 +47,18 @@ def get_comments(identifier, comment_id=None):
     # basic checks
     if identifier.startswith("T"):
         filing_model = FilingModel.get_temp_reg_filing(identifier)
-        legal_entity = LegalEntity.find_by_internal_id(filing_model.legal_entity_id)
+        business = None
+        if filing_model.legal_entity_id:
+            business = LegalEntity.find_by_internal_id(filing_model.legal_entity_id)
+        elif filing_model.alternate_name_id:
+            business = AlternateName.find_by_id(filing_model.alternate_name_id)
```

Comment:
I believe @leodube-aot is going to be adding a function to `business_service` to provide this functionality in a reusable function.  Once he has that code merged, you can rebase and use that new function here.

Reply:
updated